tsk/fs/ntfs.c

```diff
@@ -2600,11 +2608,21 @@ ntfs_proc_attrlist(NTFS_INFO * ntfs,
         /* Process the attribute seq for this MFT entry and add them
          * to the TSK_FS_META structure
          */
+        if (processed_inum_list != NULL && tsk_stack_find(processed_inum_list, mftToDo[a])) {
+            if (tsk_verbose)
+                tsk_fprintf(stderr, "ntfs_proc_attrlist: skipping previously processed MFT entry\n");
```

Comment:
I think we should set the error codes in this situation and return error.

Reply:
Do you think one of the existing error codes fits the bill or should I add a new one?

Reply:
TSK_ERR_FS_CORRUPT should work.

Reply:
Ok, I've made the change.